android/BOINC/app/src/main/assets/all_projects_list.xml

```diff
@@ -0,0 +1,933 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
```

Comment:
This file should be removed completely from here

Reply:
I remove this file